src/client.ts

```diff
@@ -115,11 +115,11 @@ export default class Client {
     log.info('finished setting up processors');
   }
 
-  boolVariation(
+  async boolVariation(
```

Comment:
It doesn't look like this function is `async`; you're not using `await`, but rather just returning the result from `this.evaluator.boolVariation`. Your options are to strip the `async` or `await` the `boolVariation` call.

Reply:
make sense